### PR TITLE
update the demo code

### PR DESCRIPTION
**Pull request by elliotchenzichang**
I ran a demo following the code marked in the file, but found that the types.StartOptions has been deprecated, so help to update the demo code.

## Description
ran the code in my local
image

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

File: content/engine/api/sdk/_index.md [hunk 1]
```diff
@@ -81,54 +81,54 @@ import (
...
-    case <-statusCh:
-    }
-
-    out, err := cli.ContainerLogs(ctx, resp.ID, types.ContainerLogsOptions{ShowStdout: true})
-    if err != nil {
-        panic(err)
-    }
-
-    stdcopy.StdCopy(os.Stdout, os.Stderr, out)
+	ctx := context.Background()
```
Comment (thaJeztah) [inline content/engine/api/sdk/_index.md:91]: For others reviewing; the large number of lines changes is due to spaces -> tabs. Go uses tabs for indenting, but let me double-check with @dvdksn if there perhaps was a reason to use spaces for the docs (maybe for formatting reasons)?

To review the changes with whitespace changes hidden, add `?w=1` to the URL; https://github.com/docker/docs/pull/20530/files?w=1

File: content/engine/api/sdk/_index.md [hunk 2]
```diff
@@ -81,54 +81,54 @@ import (
...
 	"github.com/docker/docker/pkg/stdcopy"
 )
 
 func main() {
-    ctx := context.Background()
-    cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
-    if err != nil {
-        panic(err)
-    }
-    defer cli.Close()
```
Comment (thaJeztah) [inline content/engine/api/sdk/_index.md:97]: I notice the new version has this `defer` removed; was that an intentional change?

Comment (elliotchenzichang) [reply content/engine/api/sdk/_index.md:97]: Oh, it's not intentional, add it back

**Review by dvdksn — COMMENTED** [review]
ah, this PR went below my radar for some reason. It looks like this change was already implemented in https://github.com/docker/docs/pull/21116

As for tabs vs spaces, I think it's just a formatter thing - according to the commonmark spec, tabs and spaces are both allowed. But formatters might just convert tabs to 4 spaces.

**Pull request closed without merging**